styles.css

```diff
@@ -1,31 +1,32 @@
+body {
+  background: #f7f7f9;
+  padding: 0;
+  margin: 0;
+}
+
 @media screen and (max-width: 768px) {
```

Comment:
- [ ] [OPTIONAL] I noticed that you are using a max-width media query in your CSS. Since we are designing for mobile-first, I would recommend avoiding this type of media query. Instead, please focus on using min-width media queries to progressively enhance your design as the screen size increases. Let me know if you have any questions or concerns about this. Thank you!